components/script/script_module.rs

```diff
@@ -1226,14 +1226,14 @@ pub unsafe fn EnsureModuleHooksInitialized(rt: *mut JSRuntime) {
 
 #[allow(unsafe_code)]
 unsafe extern "C" fn host_add_ref_top_level_script(value: *const Value) {
-    let val = Rc::from_raw(value);
+    let val = Rc::from_raw(value.to_private() as *const ModuleScript);
```

Comment:
Does this build? I expect you need to do `(*value).to_private()`, since it's a pointer.

Reply:
> Does this build? I expect you need to do `(*value).to_private()`, since it's a pointer.

No! I just fixed it with what you expected locally but still under building!

Reply:
Fixed again in  5830cf5 >_<